docs/devel/release-process.md

```diff
@@ -25,19 +25,19 @@ $ git pull origin
 
 Double check that the latest commit is the right one to make the release.
 
-2. Tag a release canditate version
+2. Tag a draft version, this is to ensure everything is correct as the draft release will not be published.
 
 ```bash
-$ git tag v0.x.0-rc
+$ git tag v0.x.0-draft
 ```
 
 3. Push tag to remote
 
 ```bash
-$ git push origin v0.x.0-rc
+$ git push origin v0.x.0-draft
 ```
 
-4. Verify that the CI for the tag passed
+4. Verify that the CI for the draft tag passed
 
 5. Tag the real release
```

Comment:
Not part of this PR but this step is wrong, it should be `git tag v0.x.0` instead of `git push origin v0.x.0` 🙂